app/views/organizations/organization_profiles/_form.html.erb

```diff
@@ -61,7 +46,6 @@
             <div class="alert alert-danger mt-1" role="alert"><%= error.message %></div>
           <% end %>
 
-          <%= location.label :city_town, 'City/Town' %>
```

Comment:
Same here - prefer we keep this label, too.

Reply:
Commented above